src/lib/motion_planning/VelocitySmoothing.hpp

```diff
@@ -123,6 +152,12 @@ class VelocitySmoothing
 	 */
 	static void timeSynchronization(VelocitySmoothing *traj, int n_traj);
 
+	// [m/s^2] minimum value of the smoother's maximum acceleration
+	static constexpr float kMinAccel{0.1f};
+
+	// [m/s^3] minimum value of the smoother's maximum jerk
+	static constexpr float kMinJerk{0.1f};
```

Comment:
I would keep these guards to an absolute numerical minimum. This base math library could be used with m/s or rad/us and then 0.1 might not be a good assumption anymore.

Reply:
Solved that by avoiding the division by zero rather than constraining various values at various stages to an opinionated 0.1.